test/Microsoft.DotNet.ToolPackage.Tests/ToolPackageInstallerTests.cs

```diff
@@ -101,10 +98,8 @@ public void GivenNugetConfigInstallSucceedsInTransaction(bool testMockBehaviorIs
                 TimeSpan.Zero))
             {
                 package = installer.InstallPackage(
-                    packageId: TestPackageId,
-                    packageVersion: TestPackageVersion,
-                    targetFramework: _testTargetframework,
-                    nugetConfig: nugetConfigPath);
+                    new NuGetPackageLocation(packageId: TestPackageId, packageVersion: TestPackageVersion,
```

Comment:
`NuGetPackageLocation` is just packageId packageVersion nugetConfig. One side it makes it easier to pass these 4 param around. It is also make sense to be one. These info are what nuget need to location a package